src/js/setUpSite.js

```diff
@@ -181,8 +209,16 @@ const setMapToCity = (map, cityId, cityProperties, toFitBounds = true) => {
   layer.bindPopup(popup).openPopup();
 };
 
-const setUpAutoScorecard = (map, cities) => {
-  map.on("moveend", () => {
+/**
+ * Pulls up scorecard for the city closest to the center of view.
+ * Also, loads parking lot data of any city in view.
+ *
+ * @param map: The Leaflet map instance.
+ * @param cities: Dictionary of cities with layer and scorecard info.
+ * @param parkingLayer: GeoJSON layer with parking lot data
+ */
```

Comment:
It'd be super helpful to split out refactors like this into a dedicated PR since this is such a large PR. Try to put any change that is safe to land on `main` even without lazy loading into that PR, so that this PR's diff is as minimal as possible.